crates/factor-wasi/tests/factor_test.rs

```diff
@@ -0,0 +1,37 @@
+use spin_factor_wasi::{DummyFilesMounter, WasiFactor};
```

Comment:
Nit: What do you think about the file name `wasi_test.rs` so we don't have 13 million `factor_test.rs` files? Just a thought.

Reply:
🤷